src/DynamoCoreWpf/Controls/DynamoNodeButton.cs

```diff
@@ -30,7 +30,6 @@ private DynamoViewModel DynamoViewModel
 
         public DynamoNodeButton()
         {
-            Style = (Style)SharedDictionaryManager.DynamoModernDictionary["SNodeTextButton"];
```

Comment:
Setting the style in the constructor was preventing me from overriding the style in XAML. I moved this to a default style in DynamoModern which applies to all `DynamoNodeButton`s but can be overridden in XAML.